whatshap/cli/polyphase.py

```diff
@@ -148,14 +149,15 @@ def run_polyphase(
                     out_file=output,
                     tag=tag,
                     ploidy=ploidy,
+                    include_haploid_sets=include_haploid_sets,
                 )
             )
         except OSError as e:
             raise CommandLineError(e)
 
         vcf_reader = stack.enter_context(
             VcfReader(
-                variant_file, indels=indels, phases=True, genotype_likelihoods=False, ploidy=ploidy,
```

Comment:
The trailing command should not be removed. (There are still some extraneous formatting changes in this PR; I won’t remark on the others.)